data/docs.yml

```diff
@@ -21,6 +21,10 @@
       user: apache
       repo: skywalking
       repoUrl: https://github.com/apache/skywalking.git
+      extraContributors:
+        - login: Adrian Cole
+        - login: github-actions[bot]
+        - login: dependabot[bot]
```

Comment:
Add the background of adding this, https://github.com/api7/contributor-graph/issues/146

Due to APIs limitations, we can't detect these 3 accounts(2 bots, 1 name-changed user)